src/compiler/transformers/ts.ts

```diff
@@ -2693,27 +2693,29 @@ namespace ts {
             const statements: Statement[] = [];
             startLexicalEnvironment();
 
-            let statementsLocation: TextRange;
+            let statementsLocation: TextRange | undefined;
             let blockLocation: TextRange | undefined;
             const body = node.body!;

```

Comment:
Can we remove non null assertion here given that's what you are fixing.

Reply:
I just inlined the variable.